### PR TITLE
Add comtact form

### PR DESCRIPTION
**Pull request by AHP15**
Hi, In this pull request I added the followoing:

 - Setup linters
 - Contact form
 - Styling for contact form
 
Cheers 👍

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
...
+
+      <form class="contact-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+        <input type="text" placeholder="Full name" />
+        <input type="text" placeholder="Email address"/>
+        <textarea placeholder="Enter text here"></textarea>
+
+        <button type="submit">Get in touch</button>
+      </form>
```
Comment (ABDULALI3468) [inline index.html]: Your form is not sending a POST REQUEST to the server. Please consider using the [Formspree service](https://formspree.io/html) for sending user data to the server to store it so people coming to your portfolio can interact with you efficently.

File: index.html [hunk 2]
```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+        <input type="text" placeholder="Full name" />
```
Comment (ABDULALI3468) [inline index.html]: Your Name Field should accept up to 30 characters of the name. Please consider adding the following `max-length `property to your Input Tag. Moreover, there needs a restriction so a user can't leave the Name Input empty. To achieve all the above you can use the following piece of code as an example.

`<input type="text" placeholder="Full name" maxlength="30" required/>`

File: index.html [hunk 3]
```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
...
+        <input type="text" placeholder="Email address"/>
```
Comment (ABDULALI3468) [inline index.html]: Kindly consider adding the correct type attribute to the EMAIL FIELD so users can not violate the process and you only receive the authenticated data. Moreover, there needs a restriction so a user can't leave the EMAIL Input empty.  To achieve all the above you can use the following piece of code as an example.

`<input type="email" placeholder="Email address" required/>`

File: index.html [hunk 4]
```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
...
+        <textarea placeholder="Enter text here"></textarea>
```
Comment (ABDULALI3468) [inline index.html:298]: Currently, a user can add any number of characters to the text field. Please consider adding a restriction of `max-length: 500` to restrict the number of characters. Moreover, there needs a restriction so a user can't leave the text area field empty. To achieve all this you can use the following piece of code as an example.

` <textarea placeholder="Enter text here" maxlength="500" required></textarea>`

File: css/main.css [hunk 5]
```diff
@@ -325,3 +325,53 @@ li {
...
+  height: 114px;
+  padding: 10px;
+}
+
+.final-social {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 15px;
+}
```
Comment (ABDULALI3468) [inline css/main.css:377]: Kindly review your styles and user interactions with the form. Consider making it as close with the following  [FIGMA TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=sD191wuasAZqCrMf-0)

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# `Changes Required`

### Hi @[Abdessittir](https://github.com/Abdessittir), 

> 


> 
> ## Highlights ✔️
> * [ ]  Linter checks are passing 👏
> * [ ]  Descriptive pull request title and summary 👏
> * [ ]  Descriptive readme file 👏
> 

> ## Required Changes ♻️
> I have noted a few changes that will improve your project. Please have a look at them and kindly review your project.

- [x] After making all the changes kindly submit the project for review.

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AHP15** [comment]
Thanks @ABDULALI3468, I will do the changes

File: css/main.css (hunk 5 above)
Comment (Ibizugbe) [reply css/main.css:377]: with respect to the comment of the previous reviewer, I think your project still doesn't look like the design, kindly make your project look exactly like the design as it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)😉
contact-requirement

currently, your project on desktop and mobile appears like this.
ur-desktop-contact
ur-mobile-contact
 While the design looks like this.
desktop-contact
mobile-contact

A friendly reminder😉 it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) that you use flex box for your page layout, that will help you in achieving the design look.
flexbox
also do not forget to implement the button interaction as they are part of the features in the[ design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278&t=Lq5Vdf7cIFZjtYKL-0) and it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).
btn-intr

File: index.html (hunk 4 above)
Comment (Ibizugbe) [reply index.html:298]: I can see you implemented a change on the previous reviewers comment, but you haven't implemented all the changes requested here. Kindly attend to the comment of the previous reviewer, [it is a project requirement also](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)😉
message-field
message-field2

File: index.html [hunk 6]
```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
```
Comment (Ibizugbe) [inline index.html:293]: as pointed out by the previous reviewer, It is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) that you should be able to collect data from your form with the use of the [Formspre service](https://formspree.io/html/).
currently, Your form lacks that functionality. kindly fix it😉😉
formsprre

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Abdessittir👋 , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f69656d6d596a3332456945655a59766a70712f67697068792e676966


 ## Highlights
✅Pull request title and message are meaningful.
✅The right git-flow method has been used.
✅`max-length` function has been implemented in the name input field.




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] I suggest you add a readme file to your project, the previous reviewer didn't point it, but it is [a compulsory requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). This makes your project look professional😉
.
readmeRQRR

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**